refinery/tool_manager/management/commands/load_tools.py

```diff
@@ -198,6 +213,36 @@ def _generate_workflows(self):
                     .format(workflow["name"])
                 )
 
+    def _get_available_visualization_tool_registry_names(self):
+        try:
+            response = requests.get(
+                urljoin(
+                    settings.REFINERY_VISUALIZATION_TOOL_REGISTRY_URL,
+                    "tree/{}/tool-annotations".format(
+                        self.visualization_registry_branch
+                    )
+                )
+            )
+        except requests.exceptions.RequestException as e:
+            raise CommandError(
+                "Unable to fetch Visualization Tools from the Registry "
+                "({}): {}".format(
+                    settings.REFINERY_VISUALIZATION_TOOL_REGISTRY_URL,
```

Comment:
Maybe the constant could be shorter?: `VIS_REGISTRY` might be too laconic, but gets the idea across. Your call.